andela/mmdp_site
# MMDP-731 Make env variables env independent

**What does this PR do?**

Make CMS env variables env independent:
- Use SERVER_API_URL instead of DEV_SERVER_API_URL
- Use MONGODB_URL instead of DEV_MONGODB_URL

**description of Task to be completed?**
Use env variables that are not prefixed based on the environment. This will streamline devops.

**How should this be manually tested?**
1. Update .env to have the following variables:

    ```
    JWT_SECRET=secret_key
    MONGODB_URL=mongodb://127.0.0.1/mmdp-cms
    TEST_MONGODB_URL=mongodb://localhost/mmdp_cms_test
    GMAIL_USER="your email"
    GMAIL_PASSWORD="your password"
    SERVER_API_URL=http://localhost:3000
    COOKIE_SECRET=cookie
    S3_BUCKET=""
    S3_REGION=""
    S3_SECRET=""
    S3_KEY=""
    ```

2. Update the blank ones appropriately.
3. Run the application in dev mode
    > yarn dev:server

The application should start up without any errors.

**Any background context you want to provide?**

Any pertinent information that should be considered

**What are the relevant Jira issues?**

[MMDP-731](https://viisaus.atlassian.net/browse/MMDP-731)

**Screenshots (if appropriate)**
![Screenshot 2019-07-29 at 18 32 10](https://user-images.githubusercontent.com/8180548/62061131-37bf4600-b22f-11e9-8e7c-e3a7ed63096b.png)

**Questions:**

If something is unclear or you want some questions to be addressed by your peers, mention them here
